azure-devops/azext_devops/dev/pipelines/pipeline.py

```diff
@@ -116,26 +121,38 @@ def pipeline_run(id=None, branch=None, commit_id=None, name=None, open=False, va
     :type open: bool
     :param detect: Automatically detect organization and project. Default is "on".
     :type detect: str
+    :param parameters: json string for the variables you would like to set. Example: "{'name': 'value'}"
```

Comment:
are you expecting json ? because error message is different
```
The --variables argument should consist of space separated "name=value" pairs.'
```

Reply:
For variable its Key-value pair (separated by space)